v0/webhook/transform.js

```diff
@@ -0,0 +1,35 @@
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  isURL
+} = require("../util");
+
+function process(event) {
+  const { message, destination } = event;
+  const response = defaultRequestConfig();
+  if (destination.Config.webhookUrl && isURL(destination.Config.webhookUrl)) {
+    response.endpoint = destination.Config.webhookUrl;
+    response.method = defaultPostRequestConfig.requestMethod;
+    // add content-type by default as our payload is json
+    response.headers = {
+      "Content-Type": "application/json"
+    };
+
+    const configHeaders = destination.Config.header;
+    if (configHeaders) {
+      configHeaders.forEach(header => {
+        response.headers[header.from] = header.to;
```

Comment:
Shouldn't this be header.name and header.value?

Reply:
Didn't get you completely. But I think we are not adding extra keys name and value. Kept it as it is

Reply:
Ideally these fields should be named header.name and header.value. 

For a header, _from_ and _to_ is not the right convention.

Reply:
Yes. But `from` and `to` is coming from our `config-backend` response.